datahub-web-react/src/app/ingest/source/builder/RecipeForm/TooltipWithLink.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+export const TooltipWithLink = () => {
```

Comment:
nice! since this is pretty specific to the one field, how about we call this `TableLineageModeTooltip`? Also it might make sense to just put this in the `utils` file for now - knowing that we're going to need to do some cleanup soon to figure out where everything should go in a more organized fashion.